build/utils/const.js

```diff
@@ -6,14 +6,13 @@ import path from 'path';
 export const ROOT = path.resolve(path.dirname(path.join(__dirname, '..')));
 
 export const BUILD_SYSTEM_DIR = path.join(ROOT, 'build');
-
 export const SRC_DIR = path.join(ROOT, 'app');
 export const LIB_DIR = path.join(ROOT, 'lib');
-export const APP_SHARED_DIR = path.join(SRC_DIR, 'shared');
+export const PACKAGED_DIST_DIR = path.join(ROOT, 'dist');
 
+export const APP_SHARED_DIR = path.join(SRC_DIR, 'shared');
 export const BUILD_CONFIG_PATH = path.join(SRC_DIR, 'build-config.json');
-
-export const PACKAGED_DIST_DIR = path.join(ROOT, 'dist');
+export const BROWSER_FRONTENDS_PATH = path.join(BUILD_SYSTEM_DIR, 'config', 'frontends.json');
```

Comment:
Personally, I'd find it more straightforward if instead of this pointing to a JSON file it was an array like:

    export const BROWSER_FRONTENDS = [
      'browser',
      'browser-alt',
    ];

Just because it's not going to change frequently and I don't see much value in having the extra file / JSON reading